frontend/controller/serviceworkers/sw-primary.js

```diff
@@ -135,33 +147,24 @@ sbp('okTurtles.events/on', CHELONIA_RESET, setupRootState)
       subtype: et,
       data
     }
-    self.clients.matchAll()
-      .then((clientList) => {
-        clientList.forEach((client) => {
-          client.postMessage(message, transferables)
-        })
-      })
+    broadcastMessage(message, transferables)
   })
 })
 
 sbp('okTurtles.events/on', CONTRACT_REGISTERED, (contract) => {
   // Remove function types from contract data
   // This avoids unnecessary MessagePorts
-  const argsCopy = Object.fromEntries(Object.entries(contract).filter(([_, v]) => {
-    return typeof v !== 'function'
-  }))
+  const argsCopy = {
+    manifest: contract.manifest,
+    name: contract.name
+  }
```

Comment:
This code got radically transformed... why?

And if it's safe to do it this way, then the comment on lines 155-156 no longer applies and should probably be either updated or removed.